opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -493,6 +493,8 @@ namespace Opm {
 
         const Wells* wells = wellsmanager.c_wells();
 
+        wells_ecl_ = schedule().getWells2(report_step);
```

Comment:
I really need to find out why this PR broke my branch.
Can you please tell me why this is needed here? Previously, this was only done in beginReportStep and now additionally here in initFromRestartFile

Reply:
I do not see any obvious flaw here unfortunately. What is the broken behaviour?

Reply:
Before beginReportStep was called in the restart process. Now it is not. Maybe adding the following lines to initFromRestartFile helps 

```
wells_active_ = localWellsActive() ? 1 : 0;
wells_active_ = grid.comm().max(wells_active_);
```